set.mm

```diff
@@ -471236,6 +471248,84 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
     $}
   $}
 
+  ${
+    $d A x $.  $d X x $.
+    $( The equivalence class of a cartesian product is the whole set.
+       (Contributed by Thierry Arnoux, 15-Jan-2024.) $)
+    ecxpid $p |- ( X e. A -> [ X ] ( A X. A ) = A ) $=
+      ( vx wcel cxp cec cv wbr cvv wb vex elecg mpan brxp baib bitrd eqrdv ) BA
+      DZCBAAEZFZARCGZTDZBUASHZUAADZUAIDRUBUCJCKUABSIALMUCRUDBUAAANOPQ $.
+  $}
+
+  ${
+    eqg0el.1 $e |- .~ = ( G ~QG H ) $.
+    $( Quotient group equivlance class of a subgroup.  (Contributed by Thierry
```

Comment:
equivlance -> equivalence.

Maybe better "Equivalence class of a quotient group for a subgroup."